e2e/atlas/helper_test.go

```diff
@@ -240,6 +242,18 @@ func RandProjectNameWithPrefix(prefix string) (string, error) {
 	return prefixedName, nil
 }
 
+func MongoDBMajorVersion() (string, error) {
+	atlasClient := mongodbatlas.NewClient(nil)
+	atlasURL := os.Getenv("MCLI_OPS_MANAGER_URL")
+	atlasClient.BaseURL, _ = url.Parse(atlasURL)
```

Comment:
you should not ignore this error in case `MCLI_OPS_MANAGER_URL` is not set or has a wrong value

Reply:
sorry I have missed "wrong value"

Reply:
ex: `MCLI_OPS_MANAGER_URL=wrongvalue make e2e-test`

We know it's properly set on evergreen but evergreen is not the only way to run tests